care/emr/api/viewsets/tag_config.py

```
@@ -0,0 +1,25 @@
+)
+
+
+class TagConfigViewSet(
+    EMRCreateMixin, EMRRetrieveMixin, EMRUpdateMixin, EMRListMixin, EMRBaseViewSet
+):
+    database_model = TagConfig
+    pydantic_model = TagConfigWriteSpec
+    pydantic_update_model = TagConfigUpdateSpec
+    pydantic_read_model = TagConfigReadSpec
+    pydantic_retrieve_model = TagConfigRetrieveSpec
+    # TODO AuthZ for Retrieve and Update
```

Pylint (3.3.7)

[warning] TODO AuthZ for Retrieve and Update

(W0511)